src/components/display.jsx

```diff
@@ -1,10 +1,10 @@
 import React from 'react';
 import { Mutation } from 'react-apollo';
 import { PASS } from '../graphql/mutations';
-import AuthContext from '../utils/AuthContext';
+import { AuthStoreConsumer } from '../stores/authStore';
```

Comment:
Is this a typescript naming convention? Repo/Store? I'm not so familiar. Can you explain the name change and role of these pieces?

Reply:
i just made some shit up. usually a repo is responsible for reifying models from somewhere. since i’m wrapping the context here i wanted to give it a different name, so i went with store. honestly we could call it authcontext still. that might be better.